runtimes/common/src/bonded_currencies/mod.rs

```diff
@@ -0,0 +1,305 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+pub mod hooks;
```

Comment:
Mods come usually after the uses.

Reply:
reverted in: [afffda3](https://github.com/KILTprotocol/kilt-node/pull/818/commits/afffda333d5e3253f9a6867124444d7325f922fa)

Reply:
Oh, I did that my whole live wrong. Existential crises. Ops… 

I am not sure where I changed that, but I moved the mods after the uses.